src/features/breadcrumbs/hooks/useBreadcrumbs.ts

```diff
@@ -0,0 +1,38 @@
+import { BreadcrumbElement } from 'pages/api/breadcrumbs';
+import { loadItemIfNecessary } from 'core/caching/cacheUtils';
+import { crumbsLoad, crumbsLoaded } from '../store';
+import { NextRouter, useRouter } from 'next/router';
+import { useApiClient, useAppDispatch, useAppSelector } from 'core/hooks';
+
+export default function useBreadcrumbElements() {
+  const apiClient = useApiClient();
+  const dispatch = useAppDispatch();
+  const router = useRouter();
+  const path = router.pathname;
+  const item = useAppSelector((state) => state.breadcrumbs.crumbsByPath[path]);
```

Comment:
i'm being very specific now, but I would love for this to have a more clear name to say what type of item it is!